also consider Python include directory for Python 3.x
author: Damian Alvarez (Juelich Supercomputing Centre)
updated by: Kenneth Hoste (HPC-UGent), Alex Domingo (VUB)
--- boost_1_71_0/tools/build/src/tools/python.jam.old	2019-12-19 15:11:46.626395000 +0100
+++ boost_1_71_0/tools/build/src/tools/python.jam	2019-12-19 15:12:31.441437885 +0100
@@ -544,7 +544,7 @@
     }
     else
     {
-        includes ?= $(prefix)/include/python$(version) ;
+        includes ?= $(prefix)/include/python$(version) $(prefix)/include/python$(version)m ;
 
         local lib = $(exec-prefix)/lib ;
         libraries ?= $(lib)/python$(version)/config $(lib) ;
